mayhem/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
```

Comment:
No need for a script for just one line, right? Can we put this directly in the Dockerfile?

Reply:
I don't believe so - the OSS Fuzz base image I am using exposes a method, 'compile' which runs the build script, then the compiler, as such it is proper to use the bash script here for the build stage.